docs/commands/format.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Format Command
+description: Learn more about the `format` command in Melos.
+---
+
+# Format Command
+
+<Info>Supports all [Melos filtering](/filters) flags.</Info>
+
+The format command is used to format the code in your Melos workspace according to Dart's formatting standards.
+
+```bash
+melos format
+```
+
+<Info>
+  To learn more, visit the [Dart format](https://dart.dev/tools/dart-format) documentation.
+</Info>
+
+
+## --set-exit-if-changed
+Return exit code 1 if there are any formatting changes. This flag is particularly useful in CI/CD pipelines to automatically detect and reject commits that do not adhere to the formatting standards, ensuring code quality.
```

Comment:
The max line length of the mdx files should be 80 (I missed this on the last PR)